lib/types/src/circleci.ts

```diff
@@ -22,10 +23,42 @@ export type Workflow = {
 }
 
 export interface CircleConfig {
-  version: number
-  orbs: { [orb: string]: string }
-  workflows?: {
-    version: number
-    [workflow: string]: Workflow | number
+  version: 2.1
+  orbs: {
+    [orb: string]: string
+  }
+  executors: {
+    [executor: string]: {
+      docker: { image: string }[]
+    }
+  }
+  jobs: {
+    [job: string]: {
+      docker: { image: string }[]
+      steps: (string | { [command: string]: { path?: string } })[]
```

Comment:
because the `path` field isn't necessarily used in each step in a complete CircleCI config, whereas, for instance, each `job` in a complete CircleCI config _must_ have a `steps` object. _but_ a hook doesn't need to define a `steps` object in each job, because maybe another hook defines that part

Reply:
🤯 😆 Ok I think I get it! Thank you!